app/scripts/modules/core/src/presentation/modal/Modal.module.css

```diff
@@ -145,8 +148,8 @@
 .dialog {
   display: flex;
   position: relative;
-  grid-column: 1;
-  grid-row: 1;
+  grid-column: 1 / -1;
+  grid-row: 1 / -1;
```

Comment:
🤯

Reply:
grid is WILD. IMO the best thing to come out of the standards bodies in a very very long time.